mailpoet/lib/WP/Functions.php

```diff
@@ -33,6 +33,14 @@ public function doAction($tag, ...$args) {
     return call_user_func_array('do_action', func_get_args());
   }
 
+  /**
+   * @param string $hook_name
```

Comment:
PHPStan is mad about this:

```suggestion
   * @param string $hookName
```

Reply:
Since this was a very simple change I just pushed a fix myself, 97e317d142ec6cab6ccea63cdc36ddf6b771da8d, to get the pipeline to run